sympy/parsing/c/c_parser.py

```diff
@@ -476,12 +497,18 @@ def transform_parm_decl(self, node):
             ValueError if multiple children encountered in the parameter node
 
             """
-            if (node.type.kind == cin.TypeKind.INT):
-                type = IntBaseType(String('integer'))
+            if node.type.kind in self._data_types["int"]:
+                type = self._data_types["int"][node.type.kind]
                 value = Integer(0)
```

Comment:
Can you remove the values from here too.

Reply:
Also remove it from the param definition at line 548.

Reply:
Ok